examples/docker-compose/three-servers/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+services:
+  kafka:
+    container_name: lh-kafka
+    image: bitnami/kafka:3.5
```

Comment:
Let's just use the `apache/kafka:3.8.0` one since it's now an official image, and it also means you don't need to configure any environment variables.

Reply:
Until now I didn't realize the full extent of what an Apache Kafka image release meant. Cool that they are releasing their own images now with built in env configurations!